CHANGELOG.md

```diff
@@ -81,6 +81,8 @@ Here is an overview of all new **experimental** features:
 
 ### Deprecations
 
+- **General**: Clean up previously deprecated code for 2.13 release ([#5051](https://github.com/kedacore/keda/issues/5051))
```

Comment:
Could you provide a cleared message about what has been removed? (and move this line to breaking changes section please)